pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md

```diff
@@ -0,0 +1,496 @@
+     <td>$$0$$</td>
+     <td>$$1$$</td>
+     <td style="border-left:2px solid black;">$$\frac{E_{DC}}{3}$$</td>
+     <td>$$-\frac{2E_{DC}}{3}$$</td>
+     <td style="border-right:2px solid black;">$$\frac{E_{DC}}{3}$$</td>
+</tr>
+<tr style="border-bottom:2px solid black;">
+     <td style="border-left:2px solid black;">$$7$$</td>
+     <td style="border-left:2px solid black;">$$1$$</td>
+     <td>$$1$$</td>
+     <td>$$1$$</td>
+     <td style="border-left:2px solid black;">$$0$$</td>
+     <td>$$0$$</td>
+     <td style="border-right:2px solid black;">$$0$$</td>
+</tr>
+</table>
+
+To generate the set AC voltage, the modulation block will determine in which region of the hexagon the voltage is, and it will decompose it into the 2 limiting vectors of the region. This will make possible to construct the voltage using the possible switching states. To do so, the module and angle of the voltage in the $$\alpha\beta$$ reference frame are calculated as follows:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ V_{svm} = \sqrt{V_{\alpha}^2 + V_{\beta}^2} $$
+$$ \theta_{svm} = \arctan(\frac{V_{\beta}}{V_{\alpha}}) $$
+</div>
+
+Using the appropiate duty cycle for each vector component, it is possible to reconstruct the setpoint voltage. To minimize the switching losses, each region will have a specific switching order such that only one of the IGBTs is switched at a time. For this reason, it is useful to include two options to represent the 0-valued voltage in the hexagon (vectors $$0$$ and $$7$$), in order to use the most suitable one depending on the switching state. Since all the regions have the same angles, a vector in the n-th sector can be reduced to the first sector for simplicity:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+$$ \theta_{sec1} = \theta_{svm} - \frac{\pi}{3}(n - 1)$$
+$$ n = \text{floor}(\frac{\theta_{svm}}{\frac{\pi}{3}}) + 1 $$
+</div>
+
+where the floor operation determines the sector of the hexagon depending on the angle of the voltage. The duty cycles can be calculated as:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ D_1 = \frac{\sqrt{3}V_{svm} \sin(\frac{\pi}{3} - \theta_{sec1})}{2V_{DC}} $$
+$$ D_2 = \frac{V_{svm} \sin(\theta_{sec1} - \frac{\pi}{3})}{V_{DC}} $$
+</div>
+
+where $$D_1$$ and $$D_2$$ are the duty cycles of the vectors that limit the sector. The rest of the duty times will correspond to the zero-valued vectors, which will have their times distributed as $$D_{v0} = D_{v7} = \frac{1 - D_1 - D_2}{2}$$. The total time of the PWM cycle is $$T_{PWM} = \frac{1}{f_{PWM}}$$, where $$f_{PWM}$$ is the frequency of the PWM. The time of each vector is then calculated as $$T_{v} = D_{v}T_{PWM}$$.
+
+The complete block diagram of the modulation block can be seen in the following figure: 
```

Comment:
maybe start by this diagram at the beginning to explain what we want, and then give the demonstration. 
what are q1, q2, q3?